Lib/unittest/case.py

```diff
@@ -5,6 +5,7 @@
 import difflib
 import pprint
 import re
+from typing import Iterator
```

Comment:
```suggestion
from collections.abc import Iterator
```
typing.Iterator is deprecated